### ISSUE TITLE
Give Accessible Names (`aria-label`) for All Buttons

### ISSUE THREAD

**Issue by Rinrin0413** [Sarf-Esports/website#146]
When a button doesn't have an accessible name, screen readers announce it as "button", making it unusable for users who rely on screen readers. So give the `aria-label` attributes for the button tags that do not have any accessible names.
